src/_usage/index.md

```diff
@@ -4,4 +4,6 @@ title: Usage Overview
 description: Learn how to use Quilt, and how to make the most of it.
 ---
 
-This is coming soon - watch this space!
+Hey there! Here's the usage docs for Quilt!
```

Comment:
Haha, this is too informal, but I appreciate you trying to match my done. Best thing to do here is give a paragraph explaining the section, and then an unordered list (lines starting with `*`) of links to other relevant places.

Reply:
Okay